src/Symfony/Component/ErrorHandler/Tests/Fixtures/LoggerThatSetAnErrorHandler.php

```diff
@@ -6,7 +6,7 @@
 
 class LoggerThatSetAnErrorHandler extends BufferingLogger
 {
-    public function log($level, $message, array $context = [])
+    public function log($level, $message, array $context = []): void
```

Comment:
`int $level`

Reply:
same as `BufferingLogger->log()` method, extends `vendor/psr/log/Psr/Log/LoggerInterface.php` in the end